tests/checkpointing/test_model_checkpoint.py

```diff
@@ -1270,6 +1270,77 @@ def on_load_checkpoint(self, *args, **kwargs):
     make_assertions(cb_restore, written_ckpt)
 
 
+def test_resume_training_preserves_old_ckpt_last(tmpdir):
+    # This test ensures that the last checkpoint when saved is not deleted from the previous folder
+    # (when training is resumed from the old checkpoint)
+
+    from torch.utils.data import DataLoader
+
+    from pytorch_lightning import LightningModule
+
+    class BoringModel(LightningModule):
+        def __init__(self):
+            super().__init__()
+            self.layer = torch.nn.Linear(8, 2)
+
+        def forward(self, x):
+            return self.layer(x)
+
+        def training_step(self, batch, batch_idx):
+            loss = self(batch).sum()
+            self.log("latest_is_best", self.global_step)
```

Comment:
We are logging `latest_is_best`, hence not importing it from existing helper functions.

Reply:
What do you mean?